api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -211,8 +211,8 @@ type IBKubernetesSpec struct {
 type NVIPAMSpec struct {
 	// Enable deployment of the validation webhook
 	// +kubebuilder:default:=false
-	EnableWebhook       bool `json:"enableWebhook,omitempty"`
-	ImageSpecWithConfig `json:""`
+	EnableWebhook bool `json:"enableWebhook,omitempty"`
+	ImageSpec     `json:""`
```

Comment:
k8s is OK with removing a field from CRD while there are objects with this field present right ?
what would be the behaviour ?(config field disappears ?)

Reply:
The config field disappears